fetch-perf-libs.sh

```diff
@@ -6,7 +6,7 @@ VERSION=$PERF_LIBS_VERSION-1
 set -e
 cd "$(dirname "$0")"
 
-if [[ ! -f target/perf-libs/.$VERSION ]]; then
```

Comment:
This check is broken!  froome in its cache at `/var/lib/buildkite-agent/builds/froome-1/solana-labs/solana-secondary/target/perf-libs` had the files `.v0.18.0-1` and `.v0.18.1-1`.  What happened was:
1.  `.v0.18.1-1` was installed on froome
2.  `.v0.18.0-1` was installed on froome
Now all future `.v0.18.1-1` installs claim success even though it's`.v0.18.0-1`.

A total fluke that we haven't run into this in the past.